postman_example/Enable Banking.postman_collection.json

```diff
@@ -210,6 +211,211 @@
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "https://api.enablebanking.com/payments",
+							"protocol": "https",
+							"host": [
+								"api",
+								"enablebanking",
+								"com"
+							],
+							"path": [
+								"payments"
+							]
+						}
+					},
+					"_postman_previewlanguage": "Text",
+					"header": [],
+					"cookie": [],
+					"body": "{\n    \"payment_id\": \"16e568c4-a5b4-406e-a024-7c90515031ee\",\n    \"status\": \"RCVD\",\n    \"url\": \"https://tilisy-sandbox.enablebanking.com/pis/start?payment_id=16e568c4-a5b4-406e-a024-7c90515031ee\"\n}"
+				}
+			]
+		},
+		{
+			"name": "Fetching payment",
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Authorization",
+						"value": "Bearer {{jwt_signed}}",
+						"type": "text"
+					}
+				],
+				"url": {
+					"raw": "https://api.enablebanking.com/payments/075d85d2-1683-4b77-9582-1a99f1708146",
+					"protocol": "https",
+					"host": [
+						"api",
+						"enablebanking",
+						"com"
+					],
+					"path": [
+						"payments",
+						"075d85d2-1683-4b77-9582-1a99f1708146"
+					]
+				}
+			},
+			"response": [
```

Comment:
not sure if responses add a lot of value here. maybe we could keep them, but I'm afraid the may confuse more than explain, also maintaining them require additional efforts